### PR TITLE
Biosv2

### PR DESCRIPTION
**Pull request by arsidada**

### CODE REVIEW

File: src/Components/Dashboard/Cards/EscalationsCard.js [hunk 1]
```diff
@@ -459,6 +459,7 @@ class EscalationsCard extends Component {
                     }}
                   >
                     <MenuItem aria-label="None" value="" />
+                    <MenuItem value={"coffechat"}>Account</MenuItem>
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Cards/EscalationsCard.js:462]: Should value still be coffeechat?

File: src/Components/Dashboard/Cards/CommunityCard.js [hunk 2]
```diff
@@ -158,6 +270,27 @@ class JobApplicationCard extends Component {
...
+          this.setState({
+            bio: res.data,
+            showProfile: true,
+          });
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    }
+    console.log(this.state.bio);
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Cards/CommunityCard.js:291]: Can remove

**Review by ahmedhamodi — APPROVED** [review]
Couple comments but overall lgtm

File: src/Components/Dashboard/Cards/EscalationsCard.js (hunk 1 above)
Comment (arsidada) [reply src/Components/Dashboard/Cards/EscalationsCard.js]: fixed

File: src/Components/Dashboard/Cards/CommunityCard.js (hunk 2 above)
Comment (arsidada) [reply src/Components/Dashboard/Cards/CommunityCard.js]: fixed

**Pull request merged**